src/macaron/policy_engine/prelude/helper_rules.dl

```diff
@@ -0,0 +1,61 @@
+/* Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved. */
+/* Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/. */
+
```

Comment:
Docs missing.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/46/commits/3e2695dbb0d81e9e9eb8f3c3d2f49405ccce4ac7